lib/wx/configure

```diff
@@ -3183,6 +3183,13 @@ if test "X $CFLAGS " != "X$no_werror_CFLAGS"; then
    WERRORFLAGS=-Werror
 fi
 
+case $host_os in #(
+  darwin*) :
+    enable_security_hardening_flags=no ;; #(
+  *) :
+     ;;
```

Comment:
You have to use the result as well otherwise this is a noop, 
yoou need to set CXXFALGS and/or CFLAGS.

Reply:
And there's many of them switches to modify, this feels like the change will become very large to bypass one bug, and maybe worth investing a day finding the actual bug...

Reply:
Miss understood, removed comment.